config/locales/simple_form.en.yml

```diff
@@ -4,10 +4,10 @@ en:
     hints:
       defaults:
         avatar: PNG, GIF or JPG. At most 2MB. Will be downscaled to 120x120px
-        display_name: '%{counter} characters left'
+        display_name: 'Characters left: %{counter}'
```

Comment:
You know you can do something like:

```yml
display_name:
  one: 1 character left
  other: "%{count} characters left"
```

You just have to rename counter to count.